src/packages/compiler/utils/create-boot-script.js

```diff
@@ -23,7 +23,12 @@ export default async function createBootScript(dir: string, {
         path: CWD,
         port: PORT
       })
-    );
+    ).catch(err => {
```

Comment:
This ensures that instances of `Application` will use IPC to propagate errors to the `Cluster` instance.